src/databricks/labs/ucx/queries/views/latest_historical_per_workspace.sql

```diff
@@ -0,0 +1,17 @@
+WITH last_workflow_run AS (
+    SELECT
+      workspace_id,
+      MAX(STRUCT(finished_at, workflow_run_attempt, started_at, workflow_run_id)) AS max_struct
+    FROM $inventory.workflow_runs  -- $inventory is a hardcoded name for replacing target schema in a view definition
+    WHERE workflow_name = 'migration-progress-experimental'
+    GROUP BY workspace_id
+)
+
+SELECT historical.*
```

Comment:
It's best to explicitly list the columns instead of using `.*` in this sort of context in production queries. It's tedious, I know, but it makes the queries more forward/backward compatible with respect to schema changes: otherwise new columns break aggregations, changed column order breaks unions, etc. Plus it catches dropped columns early.)

Reply:
It's a view being used by dashboards for which specific rows are filtered using a join, I think `.*` is fine here